src/public/services/AdminFormService.ts

```diff
@@ -76,3 +76,13 @@ export const deleteFormLogic = async (
     .delete(`${ADMIN_FORM_ENDPOINT}/${formId}/logic/${logicId}`)
     .then(() => true)
 }
+
+export const updateFormLogic = async (
+  formId: string,
+  logicId: string,
+  updatedLogic: ILogicSchema,
+): Promise<true> => {
+  return axios
+    .put(`${ADMIN_FORM_ENDPOINT}/${formId}/logic/${logicId}`, { updatedLogic })
+    .then(() => true)
```

Comment:
we should be returning the updated logic field here

Reply:
edited